extractor/restriction_parser.cpp

```diff
@@ -140,11 +141,11 @@ RestrictionParser::TryParse(const osmium::Relation &relation) const
     }
 
     bool is_only_restriction = false;
+    const char RESTRICTION_PREFIX[] = {"restriction"};
 
     for (auto iter = fi_begin; iter != fi_end; ++iter)

```

Comment:
You don't have to create a copy here, just use `for (; fi_begin != fi_end; ++fi_begin)`.